### DISCUSSION TITLE
EJBCA 8.0  H2 DB file location in docker container

### DISCUSSION THREAD

**Discussion by merryo11** [Keyfactor/ejbca-ce#357] [General]
Can any one guide me where does docker saves H2 DB file after the setup. I tried to save it in the file system but not able to find it.

My DB JDBC URL is jdbc:h2:/mnt/persistent/ejbcadb;DB_CLOSE_DELAY=-1

I am using Keyfactor/ejbca-ce:latest image from github. Secondly I am also stopping the docker properly using **docker stop [container name]** command for graceful shutdown.

**Comment by primetomas** [comment]
That would be in your container file system. You should be able to see where your container runtime stores things by for example "docker inspect <container-id>"

**Reply by merryo11** [reply]
my docker folder is /var/lib/docker in which I can see different folder like container,volume,overlay2,..etc but i cannot find the db file in any of these folder. See attach docker inspect file for your reference.

[docker_inspect-post.txt](https://github.com/Keyfactor/ejbca-ce/files/12714673/docker_inspect-post.txt)

**Reply by primetomas** [reply]
You have in the docker inspect the volume:
/var/lib/docker/volumes/10dac1c3cc98076f4cf52bda8478a70ae678fa65051bad8dc3afb22b11ba2682/_data

Looking on my installation I have the H2 database there (in my volume) as ejbcadb.mv.db
